sklearn/utils/validation.py

```diff
@@ -1355,6 +1377,27 @@ def check_symmetric(array, *, tol=1e-10, raise_warning=True, raise_exception=Fal
         Symmetrized version of the input array, i.e. the average of array
         and array.transpose(). If sparse, then duplicate entries are first
         summed and zeros are eliminated.
+    
+    Example
+    -------
+    >>> import numpy as np
+    >>> from sklearn.utils.validation import check_symmetric
+    >>> x = np.array([[1, 2, 2,1], [1, 2, 2,1]])
+    >>> check_symmetric(x)
+    Traceback (most recent call last):
+    File "<stdin>", line 1, in <module>
+    File "/Users/scikit-learn/scikit-learn/sklearn/utils/validation.py", line 1360, in check_symmetric
+        raise ValueError(
+    ValueError: array must be 2-dimensional and square. shape = (2, 4)
+    >>> y = np.array([[1,1], [1, 1]])
+    >>> check_symmetric(y)
+    array([[1, 1],
+       [    1, 1]])
+    >>> z = np.array([[0,1], [0, 1]])
+    >>> check_symmetric(z)
+    <stdin>:1: UserWarning: Array is not symmetric, and will be converted to symmetric by average with its transpose.
```

Comment:
please remove <stdin>:1:  ,use np.array([[0,1], [0, 1]]) directly in the method & use black formatter for linting ( line 1398 : Line too long (117 > 88))